person.rb

```diff
@@ -0,0 +1,23 @@
+require 'securerandom'
+
+class Person
+  attr_accessor :name, :age, :parent_permission
+  attr_reader :id
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @name = name
+    @age = age
+    @id = SecureRandom.uuid
+    @parent_permission = parent_permission
+  end
+
+  def can_use_service?
+    of_age || parent_permission
+  end
```

Comment:
- [ ] A variable name with a question mark (?) at the end is often used to indicate that the variable stores a boolean value or represents a condition. Kindly add **`?`** to the variable name in line 15. ℹ️ 

`of_age?`